config/shared.lua

```diff
@@ -49,4 +49,115 @@ return {
             },
         },
     },
+
+    shows = {
+        ['lsia'] = { -- location for command
+            { -- Sequences
```

Comment:
What's the benefit of having difference sequences versus one show being one sequence?

Reply:
The benefit of having different sequences just for visual separation between different segments in a show. If you really wanted to you could just have 1 sequence with a bunch of events with a longer wait time but with sequences, I feel it is easier to divide the show into different blocks.

Reply:
Couldn't the same thing be accomplished with comments? I would get rid of sequences as a data type in favor of a more simple representation of the data

Reply:
sequences have been removed, to achieve same effect just increase a wait time